src/components/EtaxForm/EtaxForm.jsx

```diff
@@ -21,12 +23,23 @@ class EtaxForm extends React.Component {
     })
   }
 
+  submitForm = () => {
+    Api.post("/address", this.state)
+    .then(response => console.log(response))
```

Comment:
Tip: You could also do `.then(console.log)` for it to be more concise. You could also use async/await instead, if you prefer.